atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -329,11 +329,7 @@ def adjust_config_kwargs(self):
         self.config_kwargs['koji_hub'] = koji_map['hub_url']
         self.config_kwargs['koji_root'] = koji_map['root_url']
 
-        odcs_fallback = {
-            'api_url': self.config_kwargs.get('odcs_url'),
-            'insecure': self.config_kwargs.get('odcs_insecure')
-        }
-        odcs_map = get_odcs(self.workflow, odcs_fallback)
+        odcs_map = get_odcs(self.workflow)
         self.config_kwargs['odcs_url'] = odcs_map['api_url']
```

Comment:
This now implies an ODCS entry is mandatory in the configmap.

We should handle the cases where ODCS integration is not enabled (this is the case for osbs-box). It would also be nice to add some unit tests to cover this case along the changed plugins.

Reply:
Good catch. Thanks. Perhaps, this piece of code shouldn't be changed. The CLOUDBLD-100 should only affect the `pre_resolve_composes`. @MartinBasti, is my understand correct?

BTW, what is the use case of  `build_orchestrate_build`, when the `config_kwargs` contains odcs config to override the corresponding odcs config defined in reactor_map_config?

Reply:
I believe that ATM, the only use case for these fallbacks is to have this chunk of code not failing when no ODCS config is found.

Since we are not using a fallback anymore, get_odcs will raise a KeyError in case odcs integration is not configured. We could set the fallback here eother to be None or to return an empty dict. Then we'd have to make sure nothing is trying to directly access the dict key. As an alternative, those fallback values could be directly set to None so we make sure these values are not being fetched from anywhere else. @rcerven WDYT?

Reply:
call get_odcs with {} as fallback
and set config_kwargs['odcs_url'] & config_kwargs['odcs_insecure'] only if they are defined